app/src/main/java/com/x8bit/bitwarden/data/platform/manager/DatabaseSchemeManagerImpl.kt

```diff
@@ -0,0 +1,17 @@
+package com.x8bit.bitwarden.data.platform.manager
+
+import com.x8bit.bitwarden.data.platform.datasource.disk.SettingsDiskSource
+import java.time.Instant
+
+/**
+ * Primary implementation of [DatabaseSchemeManager].
+ */
+class DatabaseSchemeManagerImpl(
+    val settingsDiskSource: SettingsDiskSource,
+) : DatabaseSchemeManager {
+    override var lastDatabaseSchemeChangeInstant: Instant?
```

Comment:
Since there is only one of these, is the idea that is any database has a destructive action occur, we should just resync?

Reply:
Correct. I guess it is implied. Want me to add docs somewhere explaining that?

Reply:
Sure definitely wouldn't hurt